### DISCUSSION TITLE
Is it possible to avoid auto-reconfigure on select build-variant ?

### DISCUSSION THREAD

**Discussion by t3fi** [microsoft/vscode-cmake-tools#2357] [Q&A] [answered]
Maybe didn't use the right keywords to search an answer for this, but is there a setting to avoid re-configure when selecting variant that is cached in its own folder ? I.e. when using `"cmake.buildDirectory": "${workspaceFolder}/build-${buildType}" `

BR

**Comment by bobbrow** [comment]
Sorry we missed this question earlier!

If you are using a multi-config generator and the extension is reconfiguring when you change the build type, that sounds like a bug to me and we should take a look.

**Comment by gcampbell-msft** [comment, marked as answer]
@t3fi If you still are hitting difficulties here, you can modify this behavior with the `cmake.automaticReconfigure` setting. Thanks!
